packages/experiments-realm/app-task-card.gts

```diff
@@ -186,9 +186,26 @@ class AppTaskCardIsolated extends Component<typeof AppTaskCard> {
     }
   });
 
-  @action async onMoveCardMutation(draggedCard: DndItem, newColumn: DndColumn) {
-    // This only updates the value of the card but doesn't commit to fileystem nor server
-    draggedCard.status.label = newColumn.title;
+  @action async onMoveCardMutation(draggedCard: Card, newColumn: Column) {
+    let cardUrl = new URL(draggedCard.id);
+
+    // We must fetch the specific card instance when moving cards
+    // Direct value assignment won't work without a fresh instance, it would break the modifier
+    let resource = getCard(cardUrl);
```

Comment:
We shud not use `getCard` again. bcos `getCards` already loaded the required Card